### DISCUSSION TITLE
multiple type of "VAT" (also negative) and give them name

### DISCUSSION THREAD

**Discussion by kintaro1981** [baimard/gestion#420] [Features request]
In case you are going to add multipe types of VAT please:

- make them possibile negative (like the actual VAT field).
- give the possibility to give them specific names (for example in italy in some cases you don't have VAT but "ritenuta d'acconto")

**Comment by baimard** [comment]
Taken into account, thank you for your interrest !

**Comment by digrouz** [comment]
Hello,

Did consider allowing to overwrite VAT per product and/or per Client ?

I see there several use cases:

1. A border citizen/company would want to generate "within country invoices" at regular VAT rate and "intra european" invoices without VAT
2. Some product might have different VAT rate even inside the same country.

**Reply by donaldpunk** [reply]
Sadly, that's the main reason why I can't use this app now :-( Waiting for that feature <3

**Reply by marcdasing** [reply]
> Hello,
> 
> Did consider allowing to overwrite VAT per product and/or per Client ?
> 
> I see there several use cases:
> 
>     1. A border citizen/company would want to generate "within country invoices" at regular VAT rate and "intra european" invoices without VAT
> 
>     2. Some product might have different VAT rate even inside the same country.

vat per product would be great! sometimes we have 19% and sometimes 7% (if you sell food).
this is also for me the reason why i cant use this app
thank you very much

**Comment by copctech** [comment]
Great app just a suggestion, if you could add per-line-item tax rates that would be great so if anyone wants to change the tax rate for a certain product to 0 or no tax instead of having all the items added together globally.

**Comment by baimard** [comment]
Next release include this functionnality :)

**Reply by kintaro1981** [reply]
Thank you
Please remember the possibility to give a different name too (see the first post).
